tests/Filesystem/FileTest.php

```diff
@@ -382,6 +387,9 @@ public function testMatchMimeException()
 		$this->expectException(Exception::class);
 		$this->expectExceptionMessage('Invalid mime type: text/plain');
 
+		// load translations
```

Comment:
Could you please rephrase this a little? From the code that follows it doesn't get clear why we would need the translations.

Reply:
It's for the exception message that gets checked above. I've added that to the comments.